Client/Frontend/Browser/TabManager.swift

```diff
@@ -55,20 +55,17 @@ class TabManager : NSObject {
     private let defaultNewTabRequest: NSURLRequest
     private let navDelegate: TabManagerNavDelegate
     private var configuration: WKWebViewConfiguration
-    let storage: RemoteClientsAndTabs?
+    let profile: Profile
```

Comment:
there shouldn't be as the profile has no knowledge of the TabManager, but this is probably sensible anyway.